cmd/activity-monitor/main.go

```diff
@@ -71,19 +71,7 @@ func startMonitor(rpcCh *amqp.Channel, logger *blog.AuditLogger, stats statsd.St
 		cmd.FailOnError(err, "Could not determine hostname")
 	}
 
-	err = rpcCh.ExchangeDeclare(
-		AmqpExchange,
-		AmqpExchangeType,
-		AmqpDurable,
-		AmqpDeleteUnused,
-		AmqpInternal,
-		AmqpNoWait,
-		nil)
-	if err != nil {
-		cmd.FailOnError(err, "Could not declare exchange")
-	}
-
-	_, err = rpcCh.QueueDeclare(
+	_, err = rpcCh.QueueDeclarePassive(
```

Comment:
I think it's fair to say that if by the time the totally passive Activity Monitor tries to start, if the exchange isn't already running, it's an error condition.

Taking that declaration out lets me restrict the permissions on AM down to `configure="^$" write="^$"`.